README.md

```diff
@@ -36,7 +36,7 @@ This Ruby on Rails-bsed project uses [rvm](http://rvm.io/rvm/basics) to manage a
 
 4.  Create a local config file
     ```
-    cp config/config.yml.example config/config.yml
+    cp config/exampl_econfig.yml config/config.yml
```

Comment:
I know it is the example but 
s/exampl_econfig/example_config

Reply:
Should be fixed before merge

Reply:
Fixed in 8c49d46